File: gselvacentral/WindowsApplication1/frmprincipal.vb
Imports basedato

Public Class frmprincipal
    Private cn As New mysqlbasedato("localhost", "gselvacentraldb", "root", "")
    Public nomusuario As String

    Public Sub New()
        InitializeComponent()
    End Sub

    Private Sub frmprincipal_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load, btncerrarsession.Click
        Dim a As New login(cn, Me)
        a.Show(Me)
    End Sub

    Private Sub btnseries_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnseries.Click
        Dim a As New frmseries(cn)
        a.Show()
        a.MdiParent = Me
    End Sub
    Private Sub btnadministradores_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnadministradores.Click
        Dim a As New frmadministradores(cn)
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub btncamioneros_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btncamioneros.Click
        Dim a As New frmcamioneros(cn)
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub btnunidades_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnunidades.Click
        Dim a As New frmunidades(cn)
        a.MdiParent = Me
        a.Show()
    End Sub
    Private Sub PersonasNaturalesToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles PersonasNaturalesToolStripMenuItem.Click
        Dim a As New frmclientesnaturales(cn)
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub PersonasJurídicasToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles PersonasJurídicasToolStripMenuItem.Click
        Dim a As New frmclientesjuridicos(cn)
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub btnacercade_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnacercade.Click
        Dim a As New AboutBox1
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub ToolStripButton1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ToolStripButton1.Click
        Dim a As New frmcambiarcontrasena(cn,me.nomusuario)
        a.MdiParent = Me
        a.Show()
    End Sub

    Private Sub btningresos_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btningresos.Click
        Dim a As New frmingresos(cn)
        a.MdiParent = Me
        a.Show()
    End Sub
End Class

File: gselvacentral/gselvacentral/FrmPrincipal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using basedato;

namespace gselvacentral
{
    public partial class FrmPrincipal : Form
    {
        public String nomusuario;
        private mysqlbasedato cn = new mysqlbasedato("localhost", "gselvacentraldb", "root", "");
        public FrmPrincipal()
        {
            InitializeComponent();
        }

        private void FrmPrincipal_Load(object sender, EventArgs e)
        {
            formularios.frmlogin a = new formularios.frmlogin(cn, this);
            a.Show();
        }
        /// <summary>
        /// persona jurídica
        /// </summary>
        private void toolStripButton1_Click(object sender, EventArgs e)
        {
            formularios.frmclientesjuridicos a = new formularios.frmclientesjuridicos(cn);
            a.Show();
            a.MdiParent = this;
        }
        /// <summary>
        /// persona natural
        /// </summary>
        private void toolStripButton2_Click(object sender, EventArgs e)
        {
            formularios.frmclientesnaturales a = new formularios.frmclientesnaturales(cn);
            a.Show();
            a.MdiParent = this;
        }
        /// <summary>
        /// cambiar contraseña
        /// </summary>
        private void toolStripButton4_Click(object sender, EventArgs e)
        {
            formularios.frmcabiarcontrasena a = new formularios.frmcabiarcontrasena(cn, this.nomusuario);
            a.MdiParent = this;
            a.Show();
        }

        private void toolStripButton3_Click(object sender, EventArgs e)
        {
            formularios.AboutBox1 a = new formularios.AboutBox1();
            a.MdiParent=this;
            a.Show();
        }

    }
}